pegasus/sites.v3/code.org/views/theme_fastclick.haml

```diff
@@ -1,4 +1,4 @@
-%script{:src=>"/js/fastclick.min.js"}
+%script{:async => "async", :src=>"/js/fastclick.min.js"}
```

Comment:
This will throw an error in Line 4 below (`FastClick.attach(document.body);`) when the `DOMContentLoaded` event is ever fired before `fastclick.min.js` finishes asynchronously loading/executing. This needs to be changed from `async` to `defer` to be correct.